### DISCUSSION TITLE
Hello developer, can you add this widget

### DISCUSSION THREAD

**Discussion by z369476756** [breezy-weather/breezy-weather#425] [Ideas]
![13cfdd97321c290c7ccd63c358fac32](https://github.com/breezy-weather/breezy-weather/assets/143146662/040b26a1-22fb-49ea-ad07-83e75a65001b)

**Comment by M00NJ** [comment]
This widget is already available. Go to widgets > Breezy Weather > Daily and select 'Oreo' under view style.
